modules/special-rules/pages/macro-weapon.adoc

```diff
@@ -11,16 +11,17 @@ We've borrowed the new name from the similar concept in _Epic Armageddon_.
 
 ---
 
-* In contrast to most super-heavy weapons, you may *not* choose a specific individual target for {macro-weapon} shots. 
-Instead, when you shoot with a {macro-weapon}, specify whether you will target Infantry units or Vehicle units in the target detachment.
-+[TODO: Should we rephrase to make it clear that War engine units are also valid targets, it's just that they aren't in mixed detachments with other unit types?]+
-Then allocate hits as you would for regular Firepower, but allocate only to the nearest units of the type that you specified.
-* Always roll 1 die to hit for each {macro-weapon} shot, regardless of cover.
-The shot is so powerful that it will blast straight through cover.
-* For each roll, you always need a result of 4+ to hit, regardless of the target's actual Armour value. 
+When you shoot with a {macro-weapon}:
+
+* Specify whether you will target Infantry units or Vehicle units in the target detachment.
+* For each {macro-weapon} shot, roll a D6. If you roll a 4+, the target detachment suffers a hit regardless of Armour values and regardless of cover.
+* Allocate {macro-weapon} hits as you would for regular Firepower, but allocate only to the nearest units of the type that you specified.
+
+NOTE: If shooting at a War Engine detachment, you do not need to specify a unit type.
```

Comment:
I have something like this in mind, but it's a bit different to all the others -- so I think it prudent to see what you think in case you can see flaw in the rules logic:

```suggestion
When you make one or more {macro-weapon} shots:

. Specify whether you will target Infantry units or Vehicle units in the target detachment.footnote:[To be clear, you can target a War Engine detachment, and then of course you do not need to specify a unit type as there is no mix.]
. For each shot, roll a D6.
Each 4+ that you roll is a hit -- armour and cover make no difference.
. Allocate the hits as you would for regular Firepower, but only among units of the type that you specified.
```

(As an aside, in case you're wondering: These things are somewhat subjective, but I feel that a note is a bit strong and 'noisy' for such an incidental clarification -- so I favour a less obtrusive footnote instead.)

Reply:
I've applied something close to this in my latest commit — but of course, do comment on it latest if you think that I've introduced a potential or actual issue in the rules logic.